scipy/stats/tests/test_distributions.py

```diff
@@ -2926,6 +2926,17 @@ def test_isf(self):
         assert_allclose(x, -np.log(2*p), rtol=1e-13)
 
 
+class TestLogLaplace:
+
+    def test_sf(self):
+        # reference values were computed via the reference distribution, e.g.
+        # mp.dps = 100; LogLaplace(c=c).sf(x).
+        c = np.array([2.0, 3.0, 5.0])
+        x = np.array([1e-5, 1e10, 1e15])
```

Comment:
Adjusted first test case to ensure that both branches of the `_sf` implementation are tested.